# Survey3.srv
# Request
string command
---
# Response
bool isMounted
